src/docs/sanapi/available-metrics.md

```diff
@@ -65,6 +70,49 @@ Example:
 
 If you change the `aggregation` parameter to `AVG` it will return the average transaction volume over each 7 day interval. You can see how this API can be quite powerful and flexible.
 
+## histogramData
+
+A histogram is an approximate representation of the distribution of numerical or categorical data.
+
+This is better understood with an example: For the period of January 2018 - January 2019, fetch the amount of ether that was bought in every 100-dollar price range: $0-$100, $100-$200, ... , $1300-$1400.
+
+To fetch the values for a given histogram metric, slug and time interval you can use the `histogramData` subquery of the `getMetric` API.
+
+Parameters:
+
+| Parameter | Description                                                            |
+| --------- | ---------------------------------------------------------------------- |
+| slug      | The slug of the project                                                |
+| from      | From which date to return the values in ISO 8601 format                |
+| to        | Till which date to return the values in ISO 8601 format                |
+| interval  | The intervals that should be returned. Default is `1d`, which is daily |
+| limit     | The number of results returned. Optional                               |
+
+Example:
+
+```js
+{
+  getMetric(metric: "age_distribution") {
+    histogramData(
+      slug: "santiment"
+      from: "2020-01-06T00:00:00Z"
+      to: "2020-01-07T00:00:00Z"
+      limit: 20){
+        labels
+        values{
+          ... on FloatList{
+            data
+          }
+        }
+    }
+  }
+}
+```
+
+Explanation: For all santiment tokens moved between January 6tha, 2020 and January 7th, 2020, fetch when these tokens last moved. The result is two lists - of labels and of values. For every label there is a corresponding value. The following label `The amount of tokens last moved between 2018-11-30 00:00 Etc/UTC and 2018-12-01 00:00 Etc/UTC` has a corresponding value of `18645`, meaning that 18.6k of the tokens moved in the specified 1 day interval moved for the last time between November 30th, 2018 and December 1st, 2018.
```

Comment:
Explanation: For all Santiment tokens moved between January 6th, 2020 and January 7th, 2020, fetch when these tokens last moved. The result is two lists - of labels and of values. For every label there is a corresponding value. The following label `The amount of tokens last moved between 2018-11-30 00:00 Etc/UTC and 2018-12-01 00:00 Etc/UTC` has a corresponding value of `18645`, meaning that 18.6k of the tokens moved in the specified 1 day interval moved for the last time between November 30th, 2018 and December 1st, 2018.